### PR TITLE
APPS-138: APPS-308: Added new commands AddClass, AddClassMethod, fixe…

### PR DESCRIPTION
**Pull request by vol4onok**
…d method model

### CODE REVIEW

File: src/SprykerSdk/Spryk/Model/Spryk/Builder/Method/MethodSpryk.php [hunk 1]
```diff
@@ -121,9 +121,14 @@ public function build(SprykDefinitionInterface $sprykDefinition, SprykStyleInter
     protected function methodExists(SprykDefinitionInterface $sprykDefinition): bool
     {
         $reflectionClass = $this->getReflection($sprykDefinition);
+        $className = $reflectionClass->getShortName();
```
Comment (olehnovatskyi-spryker) [inline src/SprykerSdk/Spryk/Model/Spryk/Builder/Method/MethodSpryk.php:124]: maybe $nameClass?

File: src/SprykerSdk/Spryk/Model/Spryk/Builder/Method/MethodSpryk.php [hunk 2]
```diff
@@ -121,9 +121,14 @@ public function build(SprykDefinitionInterface $sprykDefinition, SprykStyleInter
...
+        $methodName = $this->getMethodName($sprykDefinition);
```
Comment (olehnovatskyi-spryker) [inline src/SprykerSdk/Spryk/Model/Spryk/Builder/Method/MethodSpryk.php:125]: $nameMethod?

File: src/SprykerSdk/Spryk/Model/Spryk/Builder/Structure/StructureSpryk.php [hunk 3]
```diff
@@ -74,6 +75,25 @@ public function build(SprykDefinitionInterface $sprykDefinition, SprykStyleInter
         }
     }
 
+    /**
+     * @param string $path
+     *
+     * @return void
+     */
+    protected function autoloadRegister(string $path)
```
Comment (olehnovatskyi-spryker) [inline src/SprykerSdk/Spryk/Model/Spryk/Builder/Structure/StructureSpryk.php:83]: `protected function autoloadRegister(string $path): void`

File: src/SprykerSdk/Spryk/Model/Spryk/Builder/Structure/StructureSpryk.php [hunk 4]
```diff
@@ -74,6 +75,25 @@ public function build(SprykDefinitionInterface $sprykDefinition, SprykStyleInter
...
+    {
+        spl_autoload_register(function ($class) use ($path) {
```
Comment (olehnovatskyi-spryker) [inline src/SprykerSdk/Spryk/Model/Spryk/Builder/Structure/StructureSpryk.php:85]: `spl_autoload_register(function ($class) use ($path): void {`

File: tests/SprykerSdkTest/Spryk/Integration/AddClass.php [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+ * @group Integration
+ * @group AddModuleChangelogTest
+ * Add your own group annotations below this line
+ */
+class AddClass extends Unit
+{
+    /**
+     * @var \SprykerSdkTest\SprykIntegrationTester
+     */
+    protected $tester;
```
Comment (olehnovatskyi-spryker) [inline tests/SprykerSdkTest/Spryk/Integration/AddClass.php:26]: maybe add the `@property` annotation with the type into PhpDoc of the class?

File: tests/SprykerSdkTest/Spryk/Integration/AddClassMethod.php [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+ * @group Integration
+ * @group AddModuleChangelogTest
+ * Add your own group annotations below this line
+ */
+class AddClassMethod extends Unit
+{
+    /**
+     * @var \SprykerSdkTest\SprykIntegrationTester
+     */
+    protected $tester;
```
Comment (olehnovatskyi-spryker) [inline tests/SprykerSdkTest/Spryk/Integration/AddClassMethod.php:26]: `@property` in the PhpDoc of the class

**Comment by matweew** [comment]
Included into https://github.com/spryker-sdk/spryk/pull/66

**Pull request closed without merging**